istio.io/configmap-nginx.yaml

```diff
@@ -89,8 +89,11 @@ data:
           return 200 "ok\n";
         }
 
-        # Proxy to past versioned instances of the site.
-        location ~ ^/v-[0-9]+\.[0-9]+.*$ {
+        # Proxy version prefixed URL to versioned instances of
+        # https://istio.io/docs/ and corresponding css/js/image
+        location ~ ^/v-[0-9]+\.[0-9]+(/|/docs/?.*|/js/.*|/img/.*|/css/.*)?$ {
```

Comment:
given the OR clauses end with .* you don't need to put that nor the $ ? 
(modulo the corner case of v-0.1 itself (w/ w/o trailing slash))

aka isn't this a better/simpler exclude ? (also v-0.1/ does need to be sent back to root no ?)
^/v-[0-9]+\.[0-9]+(/docs/|/js/|/img/|/css/)